drivers/gpio/gpio_sifive.c

```diff
@@ -376,7 +376,7 @@ DEVICE_DT_INST_DEFINE(0,
 		    &gpio_sifive_driver);
 
 #define		IRQ_INIT(n)					\
-IRQ_CONNECT(DT_INST_IRQ_BY_IDX(0, n, irq),			\
+IRQ_CONNECT(DT_IRQN_BY_IDX(DT_DRV_INST(0), n),			\
```

Comment:
usage of `DT_DRV_INST` means there's a missing `INST` version missing

Reply:
you're right, just added a `INST` version for this